query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/converters/PinotFunctionConverter.java

```diff
@@ -78,6 +81,12 @@ private String functionToStringForDistinctCount(
     return this.config.getDistinctCountFunction(columnName) + "(" + columnName + ")";
   }
 
+  private String functionToStringForDistinctCountMv(
+      Function function, java.util.function.Function<Expression, String> argumentConverter) {
+    String columnName = argumentConverter.apply(function.getArgumentsList().get(0));
+    return this.config.getDistinctCountMvFunction(columnName) + "(" + columnName + ")";
```

Comment:
Can we use `String.format` to generate the query, just like in `functionToStringForAvgRate`?